ledger/src/blocktree_db.rs

```diff
@@ -265,6 +266,10 @@ impl Column for columns::TransactionStatus {
     const NAME: &'static str = TRANSACTION_STATUS_CF;
     type Index = (Slot, Signature);
 
+    fn key_size() -> usize {
+        std::mem::size_of::<Self::Index>()
```

Comment:
I think we can make this the default implementation of the `key_size` method on the `Column` trait.

Example of default trait method implementation here: Traits can provide default method definitions.

Reply:
(integrated, thank you!)